src/lib/helpers/runJobsQueue.js

```diff
@@ -0,0 +1,15 @@
+const os = require('os')
+
+const fastq = require('fastq')
+
+const { processPage } = require('../pages/worker')
+
+const runJobsQueue = (jobs) => {
+  console.log(`Building ${jobs.length} pages`)
+
+  const queue = fastq.promise(processPage, 4)
```

Comment:
what's the significance of 4 here?

Reply:
None! I should make it a constant. I need to make a massive site so I can benchmark it properly and find a good value.